### PR TITLE
Fix: IGV settings look ugly on some variants (#2523)

### PR DESCRIPTION
**Pull request by JessicaHillert**
This PR fixes a bug that makes IGV settings look ugly on some variants.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/variant.html [hunk 1]
```diff
@@ -475,36 +475,38 @@ <h6>No matching causative or managed variants</h6>
...
-          </tbody>
-        </table>
+      <ul class="list-group">
+        <li class="list-group-item d-flex justify-content-between">
+          <div>
+            {{ igv_button(case, variant, case_groups) }}
+          </div>
+          
+          {% if case.mt_bams or case.bam_files %}
+          {% endif %}
```
Comment (JessicaHillert) [inline scout/server/blueprints/variant/templates/variant/variant.html:485]: @northwestwitch @dnil Do we need this on line 484-485 and what does it do?

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/variant.html:485]: Looks like some leftover code that I forgot to remove last time I touched this file. You can certainly remove these lines!

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - HPO gene search button triggers any warnings for clinical / non-existing genes also on first search
 - Fixed a bug in variants pages caused by MT variants without alt_frequency
 - Tests for CADD score parsing function
+- Fixed the look of IGV settings
```
Comment (northwestwitch) [inline CHANGELOG.md:23]:
```suggestion
- Fixed the look of IGV settings on SNV variant page
```

File: scout/server/blueprints/variant/templates/variant/variant.html [hunk 3]
```diff
@@ -475,36 +475,35 @@ <h6>No matching causative or managed variants</h6>
...
-              </td>
-            </tr>
-            {% if splice_junctions_tracks %}
-              <tr>
-                <td>{{ splice_junctions_button(institute._id, case.display_name, variant._id) }}</td>
-              <tr>
-            {% endif %}
-          </tbody>
-        </table>
+      <ul class="list-group">
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant.html:478]: 👍 Looks so much better now!

**Review by northwestwitch — APPROVED** [review]
Very nice! I have just a small suggestion for the changelog entry!

**Comment by northwestwitch** [comment]
Tested on clinical-db.

[Multigene SNV variant](https://scout-stage.scilifelab.se/cust000/643594-300M/f1e8bf291eb22e2497243c4a8b22c25c) looks good:
image

image

Variant from old case with no bam file available also OK:

image

image

Cancer SNV variant also OK: https://scout-stage.scilifelab.se/cust000/coverage_test/5939d65ce239a09994ad0c34dbf453ae

Structural variant fine: https://scout-stage.scilifelab.se/cust000/643594-300M/sv/variants/bf0675db549e391d0ff5bc70b766ee37

File: CHANGELOG.md (hunk 2 above)
Comment (northwestwitch) [reply CHANGELOG.md:24]: @moedarrah do you agree with this change?

Comment (northwestwitch) [reply CHANGELOG.md:24]: Cool then I'll apply and merge. Good job both of you! 🥇

**Pull request merged**